.gitignore

```
@@ -3,3 +3,4 @@ composer.lock
 .php_cs_cache
 .phpstan_cache
 tests/.phpunit.result.cache
+.rector
```

For consistency with other files, we should probably add an end-of-file line break here.

```suggestion
.rector

```